deploy.sh

```diff
@@ -9,3 +9,5 @@ git add docs/
 git commit
 
 git subtree push --prefix docs origin gh-pages
+
+git push origin master
```

Comment:
Is it guaranteed that the person is on master branch? I think this is making some assumptions that may not be true.